client/pages/product/viewproduct.jsx

```diff
@@ -33,6 +35,11 @@ const ViewProducts = () => {
     fetchProducts();
   }, []);
 
+  const handleUpdateClick = (product) => {
```

Comment:
Didn't we discuss redirecting the user to a unique URL for the selected product? 

Explain this. With your current implementation, how can you share separate products to users via links?